main-site/components/events-calender-section/EventsCalenderSection.styles.ts

```diff
@@ -0,0 +1,203 @@
+import styled from "styled-components";
+import { max, min } from "../../../shared-ui/lib/responsive";
+import { colors } from "../../../shared-ui/style/colors";
+import { fonts, P } from '../../../shared-ui/style/typography';
+import { H3 } from '../../../shared-ui/style/typography';
+
+const StyledEventsContainer = styled.div`
+top: 3em;
+position: relative;
+`;
+
+const StyledEventsBox = styled.div`
+border-radius: 2em;
+background-color: ${colors.BUTTON_DARK_GREEN};
+width: 95%;
+position: relative;
+flex-direction: column;
+display: flex;
+margin-left: 2.5%;
+margin-top: 4%;
+margin-bottom: 2%;
+@media ${max.tabletLg} {
+  margin-top: 5%;
+  margin-bottom: 3%;
+  width: 88%;
+  border-radius: 1.7em;
+  margin-left: 2em;
+}
+@media ${max.tablet} {
+  border-radius: 1.3em;
+}
+@media ${max.tabletSm} {
+  width: 90%;
+  margin-left: 5%;
+}
+@media ${max.mobile} {
+  width: 88%;
+  margin-left: 6%;
+}
+`;
+
+const StyledTextContainer = styled.div`
+display: flex;
+margin: 4%;
+@media ${max.tablet} {
+  margin: 1.5em;
+}
+@media ${max.tabletSm} {
+  margin: 6%;
+  padding-left: 1em;
+}
+@media ${max.mobile} {
+  margin: 4%;
+  padding-left: 0.5em;
+}
+`;
+
+const TextLeftContainer = styled.div`
+width: 85%;
+`;
+
+
+const StyledEventsCalender = styled.div`
+border-radius: 2em;
+width: 50%;
+height: 40em;
+background-color: ${colors.BUTTON_GREEN};
+position: absolute;
+margin-left: 25%;
+@media ${max.tabletLg} {
+  height: 42em;
+  width: 70%;
+  margin-left: 15%;
+}  
+@media ${max.tablet} {
+  border-radius: 1.3em;
+  height: 36em;
+  margin-left: 5%;
+  width: 90%;
+}
+@media ${max.tabletSm} {
+  height: 28em;
+  margin-left: 5%;
+}
+@media ${max.mobile} {
+  height: 20em;
+} 
+`;
+
+const StyledEventsCactus = styled.img`
+width: 20em;
+right: 10em;
+top: 30em;
+position: absolute;
+
+@media ${max.tabletLg} {
+  right: 6em;
+  top: 50em;
+}  
+@media ${min.tablet} {
+  top: 50em;
+  right: 0em;
+}  
+@media ${min.tabletSm} {
+  right: 0em;
+  top: 30em;
+  height: 16em;
+}
+
+
+`;
+
+const StyledEventsGrass = styled.img`
+width: 10em;
+height: 60em;
+top: -25em;
+left: 6em;
+position: absolute;
+  @media ${min.tabletSm} {
+    left: 2em;
+    top: -30em;
+  }
+`;
+
+const EventsHeader = styled(P)`
+font-family: ${fonts.nunitoSansSemibold};
+line-height: 1.2em;
+letter-spacing: 0.08em;
+@media ${max.tabletSm} {
+  font-size: 0.8em;
+}
+@media ${max.mobile} {
+  font-size: 0.6em;
+}
+`;
+
+const EventsP = styled(P)`
+font-size: 0.8em;
+`;
+
+const EventsSubHeader = styled(P)`
+font-family: ${fonts.nunitoSansSemibold};
+font-size: 1em;
+line-height: 2em;
+@media ${max.tabletSm} {
+  font-size: 0.8em;
+}
+@media ${max.mobile} {
+  font-size: 0.5em;
+}
+`;
+
+const StyledH3 = styled(H3)`
+text-align: center;
+letter-spacing: 0.2em;
+`;
+
+const EventsPBolded = styled(P)`
+font-family: ${fonts.nunitoSansSemibold};
+font-size: 0.8em;
+
+`;
+
+const EventsLocationP = styled(P)`
```

Comment:
For the EventsLocationP, right now it's not consistently at the bottom and also isn't always 2 lines when it could be. You can remove all of this styling and make it position absolute then style the bottom + right. (See video). You might have to tweak some of the widths a tiny bit on the smaller screen sizes but it should reduce the amount of code here by a good amount since u don't have to keep changing the padding-top at every screensize.
https://user-images.githubusercontent.com/59738880/194759572-90ec69a8-8dc2-43e9-8131-bf707323a952.mov

Reply:
Oop i think in the video I added the width to just the P tag with the long text that was too close to the location but I think it'd be better to add the width to the container with all of the text in it (besides the location) 

<img width="694" alt="Screen Shot 2022-10-09 at 9 37 09 AM" src="https://user-images.githubusercontent.com/59738880/194760000-bcb3c721-eeee-48b4-8aca-fec43747a0f7.png">